spec/features/saml/saml_spec.rb

```diff
@@ -505,6 +505,22 @@
         complete_idv_questions_ok
         expect(current_url).to eq saml_authn_request
       end
+
+      it 'contains verified profile attributes in SAML Response' do
+        saml_authn_request = loa3_authnrequest
+        visit saml_authn_request
+        visit new_user_registration_path
+        sign_up_and_2fa
+        click_on 'Continue'
+        fill_out_idv_form_ok
+        click_button 'Continue'
+        complete_idv_questions_ok
+
+        saml_response_assertions = SamlResponseHelper::XmlDoc.new('feature', 'response_assertion')
+
+        expect(saml_response_assertions.attribute_value_for('first_name')).to eq 'Some'
+        expect(saml_response_assertions.attribute_value_for('last_name')).to eq 'One'
+      end
```

Comment:
Is it possible to use a controller spec for this? It would be faster than going through the account creation and IdV process in the browser, since those seem to only be required as a setup for this test.